backend/test/participant/participant_update/UpdateExistentParticipant.test.js

```diff
@@ -0,0 +1,22 @@
+const { expect } = require('chai');
+const fetch = require('node-fetch');
+
+const SERVER = 'http://localhost:8000/api';
+
+describe('UpdateNonExistentParticipant', () => {
```

Comment:
Please change the name of this file to `UpdateExistingParticipant.test.js` and change the contents so that it tests a valid 'update'.

Or, remove this file from the PR and add it in a subsequent PR.